packages/table/src/quadrants/tableQuadrantStack.tsx

```diff
@@ -83,16 +86,41 @@ export interface ITableQuadrantStackProps extends IProps {
     isVerticalScrollDisabled?: boolean;
 
     /**
-     * The number of frozen columns.
+     * A list of `TableLoadingOption`. Loading cells may have different sizes
+     * from potentially custom cells in the header or body, so we need to know
+     * when the loading states change in order to synchronize quadrant sizes
+     * properly.
+     */
+    loadingOptions?: TableLoadingOption[];
+
+    /**
+     * The number of columns. Affects the layout of the table, so we need to
+     * know when this changes change in order to synchronize quadrant sizes
```

Comment:
"changes change" in every comment all the way down

Reply:
👍 👍